lib/membrane_sdl/player.ex

```diff
@@ -16,7 +16,10 @@ defmodule Membrane.SDL.Player do
   # The measured latency needed to show a frame on a screen.
   @latency 20 |> Time.milliseconds()
 
-  def_input_pad :input, accepted_format: RawVideo, flow_control: :manual, demand_unit: :buffers
+  def_input_pad :input,
+    accepted_format: %RawVideo{pixel_format: format} when format != :RGB,
```

Comment:
I'm quite sure that nothing but `I420` will work properly

```suggestion
    accepted_format: %RawVideo{pixel_format: :I420},
```